mam/api/tests/test_api.c

```diff
@@ -369,6 +398,7 @@ static void test_api_multiple_packets_run(mam_api_t *const param_api, size_t con
   tryte_t *payload_out = NULL;
   size_t payload_out_size = 0;
   bool is_last_packet;
+  size_t remaining_sks;
```

Comment:
considering the coding style in this file, how about moving it into the for loop?

Reply:
@oopsmonk Addresses, thanks!